cmd/postpolicyform.go

```diff
@@ -262,7 +262,7 @@ func checkPolicyCond(op string, input1, input2 string) bool {
 // (http://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-HTTPPOSTConstructPolicy.html)
 func checkPostPolicy(formValues http.Header, postPolicyForm PostPolicyForm) error {
 	// Check if policy document expiry date is still not reached
-	if !postPolicyForm.Expiration.After(UTCNow()) {
+	if !postPolicyForm.Expiration.UTC().After(UTCNow()) {
```

Comment:
No need. "After" handles timezones fine.
```suggestion
	if !postPolicyForm.Expiration.After(UTCNow()) {
```

Reply:
I think I did this to fix a bug that I found with Expiration when it is TZ specific.

Reply:
The conversion will not do anything, since the absolute time will be the same.